cdap-cli-tests/src/test/java/co/cask/cdap/cli/CLIMainTest.java

```diff
@@ -519,6 +519,38 @@ public void testRuntimeArgs() throws Exception {
   }
 
   @Test
+  public void testVersionedRuntimeArgs() throws Exception {
+    String versionedServiceId = String.format("%s.%s version %s", FakeApp.NAME, PrefixedEchoHandler.NAME, V1_SNAPSHOT);
```

Comment:
Maybe just create a private method that takes parameter `String serviceId` and `ServiceId service` so that `testRuntimeArgs()` and `testVersionedRuntimeArgs()` can both call this method without duplicate code.